README.md

```diff
@@ -29,6 +29,11 @@ $ cmake -G <generator> [options] ../benchmark
 $ make
 ```
 
+Similarly for the Visual Studio 15 2017 generator we can use
+```bash
+> msbuild benchmark.sln /p:Configuration=Release /p:Platform=Win32
```

Comment:
Do we want to spell it out for every cmake generator?
Maybe let's just
```diff
-$ make
+$ cmake --build ./
```
that is more correct than hardcoding `make` there anyway